include/mbedtls/config.h

```diff
@@ -2996,6 +3037,27 @@
 //#define MBEDTLS_PLATFORM_NV_SEED_READ_MACRO   mbedtls_platform_std_nv_seed_read /**< Default nv_seed_read function to use, can be undefined */
 //#define MBEDTLS_PLATFORM_NV_SEED_WRITE_MACRO  mbedtls_platform_std_nv_seed_write /**< Default nv_seed_write function to use, can be undefined */
 
+/**
+ * \brief       This macro is invoked by the library when an invalid parameter
+ *              is detected that is only checked with MBEDTLS_CHECK_PARAMS
+ *              (see the documentation of that option for context).
+ *
+ *              When you leave this undefined here, a default definition is
+ *              provided that invokes the function mbedtls_param_failed(),
+ *              which is declared in platform_util.h for the benefit of the
+ *              library, but that you need to define in your application.
```

Comment:
I would add "yourself" to the end of this sentence.

Reply:
I think the "you" and "your" in the sentence are already explicit enough, so not changing that.

I'm leaving the "here" because I don't want to "leave this undefined" to be confusing: obviously the macro will be defined _somewhere_, just not in this file.